cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.tsx

```diff
@@ -33,15 +39,66 @@ const EdgeInfoContent: FC<{ selectedEdge: NonNullable<SelectedEdge> }> = ({ sele
                 <>
                     {Object.entries(sections).map((section, index) => {
                         const Section = section[1];
+
+                        let handleOnChange = (label: string, isOpen: boolean) => {};
+
+                        if (
+                            (selectedEdge.name === 'GoldenCert' || selectedEdge.name === 'ADCSESC1') &&
+                            section[0] === 'composition'
+                        ) {
```

Comment:
I think this might read a bit better if the conditional logic is moved into the prop on `EdgeInfoCollapsibleSection` -- so instead of conditionally changing the function definition, make `onChange` optional and conditionally set it to `handleOnChange`

Reply:
I gave this a try. Let me know if that is what you had in mind or if we can change this to make it clearer still